torch/csrc/jit/codegen/cuda/python_frontend/python_bindings.cpp

```diff
@@ -630,9 +633,27 @@ void initNvFuserPythonBindings(PyObject* module) {
               broadcast_dims[idx] < static_cast<int>(output_shape.size()),
               "Invalid broadcast_dims value.");
           is_broadcast_dim.at(broadcast_dims[idx]) = false;
+          is_expand_dim.at(broadcast_dims[idx]) =
```

Comment:
Question: This is trying to capture that the expand operation needs to expand a previously existing dimension that was already size 1, correct?  If that is the case, could you add a comment?